user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -368,6 +368,73 @@ Caused by: org.opensearch.OpenSearchException: Transport client authentication n
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) [?:?]
+  Caused by: org.opensearch.transport.RemoteTransportException: [DataNodeName][166.206.186.147:9300][internal:cluster/coordination/join/validate]
+  Caused by: org.opensearch.cluster.coordination.CoordinationStateRejectedException: join validation on cluster state with a different cluster uuid tNh7sXJeQjuf-RTTfFt7qg than local cluster uuid qVI2q9lMSy-Ot7O9v68d_A, rejecting
+          at org.opensearch.cluster.coordination.JoinHelper.lambda$new$4(JoinHelper.java:219) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.indexmanagement.rollup.interceptor.RollupInterceptor$interceptHandler$1.messageReceived(RollupInterceptor.kt:113) ~[?:?]
+          at org.opensearch.performanceanalyzer.transport.PerformanceAnalyzerTransportRequestHandler.messageReceived(PerformanceAnalyzerTransportRequestHandler.java:43) ~[?:?]
+          at org.opensearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:106) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.transport.InboundHandler$RequestHandler.doRun(InboundHandler.java:453) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:806) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:52) ~[opensearch-2.8.0.jar:2.8.0]
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) ~[?:?]
+  ```
+
+To fix this issue:
+
+1. Stop OpenSearch on the cluster manager and data nodes.
+
+1. Check the *cluster.initial_master_nodes* and *discovery.seed_hosts* on all nodes for any differences.
+
+1. Go to the data folder specified in the *opensearch.yml* file and delete the folder called *nodes* along with every file in it by using the following command:
+
+   ```bash
+   sudo rm -rf nodes
+   ```
+
+1. Restart OpenSearch on the cluster manager first and then on the data nodes.
+
+> [!IMPORTANT]
+> Deleting the nodes folder will result in loss of data. You should only do so with a new installation of OpenSearch.
+
 ### SLSearch.txt logging mentions OpenSearch version is not officially supported
 
-If the SLSearch.txt log file mentions that the OpenSearch version is not officially supported, upgrade your DMS to DataMiner 10.3.6 or higher.
+If the *SLSearch.txt* log file mentions that the OpenSearch version is not officially supported, upgrade your DMS to DataMiner 10.3.6 or higher.
```

Comment:
maybe we should mention here that this has no functional impact, the DMA will run fine either way. Just a rogue log line